BankSDK/GiniBankSDK/Sources/GiniBankSDK/Core/ReturnAssistant/EditLineItem/Components/QuantityView.swift

```diff
@@ -57,7 +57,7 @@ final class QuantityView: UIView {
         button.setImage(prefferedImage(named: "quantity_plus_icon"), for: .normal)
         button.addTarget(self, action: #selector(increaseQuantity), for: .touchUpInside)
         button.translatesAutoresizingMaskIntoConstraints = false
-        let descriptor = NSLocalizedStringPreferredFormat("ginibank.digitalinvoice.edit.plus.button.accessibility",
+        let descriptor = NSLocalizedStringPreferredGiniBankFormat("ginibank.digitalinvoice.edit.plus.button.accessibility",
                                                           comment: "Increase quantity")
```

Comment:
[Suggestion] same suggestion: a indentation would be nice here, to align `comment` parameter in the right place